department-of-veterans-affairs/caseflow
# Your Correspondence - In Progress Tab (Bug fixes)

Resolves [Your Correspondence - In Progress Tab](https://jira.devops.va.gov/browse/APPEALS-35479)

# Description
Previously, when the user filtered by multiple task types, the filter results would break. These code changes fix this issue and also update some code within the changed filter object to be OOP best practice compliant. This also fixes the eFolderUploadFailedTask redirecting to the wrong area.

## Acceptance Criteria

## Testing Plan

1. Login as JOLLY_POSTMAN
2. Navigate to your correspondences
3. Toggle any of the task type filters. Note that the table result count now reflects the number indicated by the task dropdown.
4. Toggle an additional task type, it will now reflect previous + current task items count.

### Code Climate
Your code does not add any new code climate offenses? If so why?
